.github/workflows/image-build.yml

```diff
@@ -12,15 +16,33 @@ jobs:
   docker-image:
     name: Check docker image build
     runs-on: ubuntu-latest
+    env:
+      IMAGE_NAME: stacklok/codegate
+      IMAGE_TAG: dev
```

Comment:
We don't push this image do we? It's just a tag used for the artifact that we upload and then download?

Reply:
We don't, no. This is just so it has a reference that we can then use to save it as a tar file.